### PR TITLE
fix: building documentation up to date for 18.04, tensorrt and tests

### PR DESCRIPTION
**Pull request by beniz**

### CODE REVIEW

File: docs/source.md [hunk 1]
```diff
@@ -4,7 +4,31 @@ Below are instructions for 18.04 LTS. For other Linux and Unix systems, steps ma
...
+
+With CUDA 10, a more recent version of cmake than that of Ubuntu 18.04 is required, and needs to be installed beforehand:
+
+```
+wget https://github.com/Kitware/CMake/releases/download/v3.14.0/cmake-3.14.0.tar.gz
+tar xvzf cmake-3.14.0.tar.gz
+cd cmake-3.14.0
+./bootstrap
+make
+sudo make install
```
Comment (sileht) [inline docs/source.md:31]: We should recommend the Debian packages:  https://apt.kitware.com/

Comment (sileht) [reply docs/source.md:31]: DevOps often cry when they see `sudo make install`

Comment (beniz) [reply docs/source.md:31]: OK, what do I need to change exactly ?

Comment (sileht) [reply docs/source.md:31]:
```
sudo apt-get install apt-transport-https ca-certificates gnupg software-properties-common wget
wget -q -O- https://apt.kitware.com/keys/kitware-archive-latest.asc | sudo apt-key add -
sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
sudo apt-get update
sudo apt-get install cmake kitware-archive-keyring
```

Comment (beniz) [reply docs/source.md]: done

**Pull request merged**